.github/workflows/packaging.yaml

```diff
@@ -94,3 +94,27 @@ jobs:
         run: |
           ./packaging/msi/winget_manifest.sh > Tetrate.func-e.yaml
           yamllint -sd '{extends: default, rules: {line-length: disable}}' Tetrate.func-e.yaml
+
+      # This only checks installing the Debian package (amd64) when running this job step on Linux
+      # (Ubuntu, amd64) as it is more efficient than having a new job definition (which requires to
+      # upload and download the built packages as artifacts) and do the verification there.
+      - name: "Test installing Debian package (Ubuntu, amd64)"
```

Comment:
Following the footsteps of testing Windows .msi here. Not sure if the reasoning (hence the decision to add it here) is correct.